test/helpers/kubectl.go

```diff
@@ -18,6 +18,7 @@ import (
 	"bytes"
 	"context"
 	"encoding/json"
+	"errors"
```

Comment:
Looks like this one is unintentional, right?

Reply:
It was used for initiating the error. But isn't anymore.